modoptions.lua

```diff
@@ -382,6 +382,18 @@ local options={
 		step   = 0.1,
 		section= "raptor_defense_options",
 	},
+	{
+		key    = "raptor_eggresourcemult",
+		name   = "Egg Resource Reclaim Multiplier (Range: 0.1 - 3)",
```

Comment:
says 3 but actually is 10
```suggestion
		name   = "Egg Resource Reclaim Multiplier (Range: 0.1 - 10)",
```

Reply:
fixed ty